src/providers/node.rs

```diff
@@ -7,12 +7,12 @@ use crate::nixpacks::{
     nix::Pkg,
     phase::{BuildPhase, InstallPhase, SetupPhase, StartPhase},
 };
-use anyhow::{bail, Result};
+use anyhow::Result;
 use regex::Regex;
 use serde::{Deserialize, Serialize};
 
+const DEFAULT_NODE_PKG_NAME: &'static &str = &"nodejs";
```

Comment:
```suggestion
const DEFAULT_NODE_PKG_NAME: &'static &str = &"nodejs-16_x";
```

Reply:
Not needed

Reply:
I believe this is required. If nix updates default version it will break nixpack's reproducibility.